### PR TITLE
`RPA.HTTP`: Update readme with additional information and add examples

### PR DESCRIPTION
**Pull request by blakewaud**
* Add link in library Readme to the RequestsLibrary keyword documentation page.
* Explain in library Readme what is added by our extension.
* Add examples to our keywords on usage.

### CODE REVIEW

**Review by cmin764 — CHANGES_REQUESTED** [review]
LGTM! Just a concern around the `Download` keyword docstring RF example.

File: packages/main/src/RPA/HTTP.py [hunk 1]
```diff
@@ -173,6 +202,25 @@ def download(
...
+            ${DOWNLOAD_PATH}=   ${OUTPUT DIR}${/}downloads
+            ${WORD_EXAMPLE}=    https://file-examples.com/wp-content/uploads/2017/02/file-sample_100kB.doc
+            ${EXCEL_EXAMPLE}=   https://file-examples.com/wp-content/uploads/2017/02/file_example_XLS_10.xls
+
+            *** Tasks ***
+            Download files with reused session with provided file names
+                # Starts a new session
+                HTTP Get    ${WORD_EXAMPLE}    target_file=${DOWNLOAD_PATH}
+                # Uses the previous session
+                HTTP Get    ${EXCEL_EXAMPLE}    target_file=${DOWNLOAD_PATH}
```
[the remark this answers is not in the document]
Comment (blakewaud) [reply packages/main/src/RPA/HTTP.py:220]: Whaps! Yeah totally forgot to change the kw when I copied the example, thank you!

File: packages/main/src/RPA/HTTP.py [hunk 2]
```diff
@@ -110,7 +139,7 @@ def http_get(
             the target file, default ``False``
         :param stream: if ``False``, the response content will be immediately downloaded
         :return: request response as a dict
-        """
+        """  # noqa: E501
```
[the remark this answers is not in the document]
Comment (blakewaud) [reply packages/main/src/RPA/HTTP.py:142]: I'm thinking this is it's own issue and one that I've been wanting to address for awhile, honestly. I will open this up and it probably should be addressed as part of the other documentation updates #318 and #357.

**Review by cmin764 — APPROVED** [review]

File: packages/main/src/RPA/HTTP.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (cmin764) [reply packages/main/src/RPA/HTTP.py:142]: Sounds good, thx!

**Pull request merged**